apps/src/templates/progress/ProgressPill.story.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ProgressPill from './ProgressPill';
+import { LevelStatus } from '@cdo/apps/util/sharedConstants';
+
+export default storybook => {
+  storybook
+    .storiesOf('ProgressPill', module)
+    .addStoryTable([
```

Comment:
I wonder if we should write a tiny helper that would let us run these stories as unit tests, at least catching that they render without errors and giving us the coverage.  Should be pretty easy with enzyme.